### ISSUE TITLE
A11y_Logic Apps VS extension - Stay Healthy_Add New connection: No error identification messages are provided when user the mandatory fields "Connection Name, Connection String" are left empty

### ISSUE THREAD

**Issue by InduPriya1805** [facebook/draft-js#2206]
**Environmental Details:**
Microsoft Visual Studio Enterprise 2019 Preview​
Version 16.2.0 Preview 3.0​
Microsoft Windows Version 1903 (OS Build 18362)​
​
**Note:​
Similar issue is observed in HTTP at URL field**
​
**Repro Steps​**
1)Open Visual Studio 2019 preview latest version
2)Create a new project under Azure Resource group​
3)Select azure template as 'Logic App' and create solution​
4)Tab till Extensions->Manage extensions->Online and search for logic app and install and download 'Azure logic apps tools for Visual studio 2019'​
5)Navigate till LogicApp.json file and right click->Open with logic app designer​
6)Provide the subscription and resource group details and press enter​
7)Tab till Service Bus queue trigger and press enter​
8)Tab till Add New connection-Browse/Manual and press 
9) Verify user is getting any error identification messages when the mandatory fields "Connection Name, Connection String" are left empty or not

**​Actual Result:​**
No error identification messages are provided when user the mandatory fields "Connection Name, Connection String" are left empty
​
**Expected Result:​**
Error identification messages should be provided when user the mandatory fields "Connection Name, Connection String" are left empty
​
**User Impact:​**
Users with disability will get to understand about the error clearly only once some error messages are provided 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={733120d0-56c5-479c-889f-fee36ea08ee0}

**Comment by InduPriya1805** [comment]
Can you please provide any update on the bug?

**Comment by InduPriya1805** [comment]
Can you please provide any update on the bug?

**Comment by joechung-msft** [comment]
Duplicate of #1696
